src/go/configgenerator/filterconfig/filter_generator.go

```diff
@@ -194,10 +273,18 @@ func makeTranscoderFilter(serviceInfo *ci.ServiceInfo) *hcmpb.HttpFilter {
 					PreserveProtoFieldNames:    serviceInfo.Options.TranscodingPreserveProtoFieldNames,
 				},
 			}
-			if serviceInfo.Options.TranscodingFilePath != "" {
+			if serviceInfo.Options.TranscodingFilePathHost != "" {
 				transcodeConfig.DescriptorSet = &transcoderpb.GrpcJsonTranscoder_ProtoDescriptor{
-					ProtoDescriptor: serviceInfo.Options.TranscodingFilePath,
+					ProtoDescriptor: serviceInfo.Options.TranscodingFilePathHost,
+				}
+			}
+			if serviceInfo.Options.TranscodingFilePathOutput != "" {
```

Comment:
This is only needed by asm gen? Can we do it there in google3? I feel this side effect is too subtle for makeTranscoderFilter which is a bit hard to find.

Reply:
I will take a look.  The idea is after TranscoderFilter is generated,  ASM code needs to Unmarshal it and modify it according to its special need.   I will see how much is the code changes,   if it is too big,  will do it in the separate PR.